t/150-lazy-start.t

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env tarantool
+local tap = require('tap')
+local tnt = require('t.tnt')
+
+local test = tap.test('test driver register')
+test:plan(3)
+
+local function check_lazy_start()
+    -- Needed for bootstrap
+    tnt.cfg{}
+
+    tnt.cfg{read_only = true}
+    local queue = require('queue')
+
+    local res, err = pcall(function() queue.stats() end)
+    local check = not res and
+        string.match(err, 'Please run box.cfg{} first') ~= nil
```

Comment:
It looks a bit misleading: box is already configured here, but in read-only. Maybe it worth to adjust the error message (it is from queue/init.lua).

Reply:
Updated